### PR TITLE
fix: 404 links in #10

### PR DESCRIPTION
**Pull request by lucasgonze**
Fixes broken links in https://github.com/newrelic/developer-toolkit-template-go/issues/10

I unlinked the link text rather than correcting the URLs. That is because this repository is a template, and things are more likely than usual to move around.

### CODE REVIEW

**Comment by lucasgonze** [comment]
The security issue appears to be a CI problem. It needs the Snyk API call to be authenticated. I will get a sign-off from another admin, then force-merge.

**Comment by lucasgonze** [comment]
@elaguerta This trivial PR is blocked on a spurious Snyk failure. I propose to force merge it. Thoughts?

**Comment by elaguerta** [comment]
@lucasgonze I agree, please merge over the checks. These changes are only to markdown, and there aren't any secrets in the new text, so we don't need to run Snyk on this PR. Thanks for checking in!

**Review by sanderblue — APPROVED** [review]

**Pull request merged**
